src/chunk.c

```diff
@@ -2766,18 +2766,29 @@ init_scan_by_compressed_chunk_id(ScanIterator *iterator, int32 compressed_chunk_
 }
 
 bool
-ts_chunk_contains_compressed_data(Chunk *chunk)
+ts_chunk_contains_compressed_data(Chunk *chunk, Chunk **parent)
 {
 	ScanIterator iterator = ts_scan_iterator_create(CHUNK, AccessShareLock, CurrentMemoryContext);
-	bool found = false;
+	Datum parent_id = 0;
```

Comment:
Is this a reliable and supported way of indicating a non-set Datum? I mean, it seems 0 is a valid value  for ,e.g., int datums.